src/data/corpus.cpp

```diff
@@ -1,3 +1,8 @@
+/* Part of this file was contributed by NVIDIA under license:
+ *   Copyright (C) 2020 NVIDIA Corporation
+ *   SPDX-License-Identifier: MIT
```

Comment:
What is this for? It seems the only change is the rounding of `maxDims`. What NVidia contribution was made here?

In general, I would be opposed to changing the comment style for licence. Marian source code does not have license information in the source files directly, but rather in a separate license file. Please let's continue to follow that pattern.

Reply:
Yes that was the only contribution. However, I was told to include a notice even in files where I make one line changes. This is just me following instructions.

Reply:
Also, I just saw the second part of your comment. Is there a process for adding NVIDIA to the license file? I'm not sure what the solution is here.

Reply:
The one that hasn't been updated since 2016 and doesn't even name Microsoft?  I guess add it to your PR and shame Marcin.

Reply:
@kpu I think this is the one Frank was referring to. I'll ask Marcin if this is ok. I will also need to check to see if we are internally ok with removing the notices assuming we are added to the license file.

@emjotde What do you suggest as the way forward?

Reply:
I checked internally and I can add NVIDIA to the main license file and remove the notices in all the other files! 

I will take care of this in all the PRs I have submitted.